src/controller/middleware/auth.rs

```diff
@@ -71,7 +75,15 @@ where
             Ok(claims) => {
                 let user = T::find_by_claims_key(&ctx.db, &claims.claims.pid)
                     .await
-                    .map_err(|_| Error::Unauthorized("token is not valid".to_string()))?;
+                    .map_err(|e| {
+                        match e {
+                            ModelError::EntityNotFound => {
+                                Error::Unauthorized("not found".to_string())
+                            }
+                            ModelError::DbErr(_) => Error::InternalServerError, // don't show 401 for system errors
+                            _ => Error::Unauthorized(format!("other error: '{e}'")),
```

Comment:
* let log the error message to tracing::error..
* Chaneg the other error to `could not authorized`